src/app/birthday/birthday.service.spec.ts

```diff
@@ -10,7 +10,34 @@ describe('Service: Birthday', () => {
     });
   });
 
-  it('should ...', inject([BirthdayService], (service: BirthdayService) => {
-    expect(service).toBeTruthy();
-  }));
+  describe('getBirthdayFor() method', () => {
+    it('should return new Date observable', fakeAsync(() => {
+      const callback = jasmine.createSpy();
+
+      getService()
+          .getBirthdayFor('Vasya')
+          .subscribe(callback);
+
+      tick(5000);
+
+      expect(callback).toHaveBeenCalledWith(jasmine.any(Date));
+    }));
+
+    it('should return error observable', fakeAsync(() => {
```

Comment:
The test itself is correct but the spec name is misleading - you are expecting the whole function to throw - not the observable to be returned with error.